tests/js/server/singleTemplateRouter-tests.js

```diff
@@ -0,0 +1,187 @@
+// Tests for the "single template router".
+//
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("../../../index");
+require("../lib/promiseWrapper");
+
+var jqUnit = require("node-jqunit");
+var fs     = require("fs");
+var path   = require("path");
+var jsdom  = require("jsdom");
+
+var jqueryPath = path.resolve(__dirname, "../../../node_modules/infusion/src/lib/jquery/core/js/jquery.js");
+var jqueryContent = fs.readFileSync(jqueryPath, "utf8");
+
+var express = require("gpii-express");
```

Comment:
See previous notes about use of "express" global variable. We already have the gpii namespace active, so the next line can be "gpii.express.loadTestingSupport()"